source/js/main.js

```diff
@@ -493,19 +483,22 @@ let main = {
     }
 
     // store profile cards
-    let profileCards = document.querySelectorAll(`.profiles .person-card`);
+    function getProfileCards() {
+      return document.querySelectorAll(`.profiles .person-card`);
+    }
 
-    // checks for profile cards in the initial page load
-    if (profileCards.length > 0) {
-      bindProfileCardAnalytics(profileCards);
+    function updateProfileList() {
+      let profileCards = getProfileCards();
+      if (profileCards.length > 0) {
+        bindProfileCardAnalytics(profileCards);
+      }
     }
-    // And start listening for profile filter events,
-    // in case profile cards get updated.
-    document.addEventListener(`profiles:list-updated`, () => {
-      // Refetch the profile cards, because they'll have gone stale.
-      profileCards = document.querySelectorAll(`.profiles .person-card`);
-      bindProfileCardAnalytics(profileCards);
-    });
+
+    document.addEventListener(`profiles:list-updated`, () =>
+      updateProfileList()
+    );
+
+    updateProfileList();
```

Comment:
I simplied this chunk of code further + added back the inline comments
```suggestion
function updateProfileList() {
  let profileCards = document.querySelectorAll(`.profiles .person-card`);
  bindProfileCardAnalytics(profileCards);
}

// Checks for profile cards in the initial page load
updateProfileList();

// And start listening for profile filter events,
// in case profile cards get updated.
document.addEventListener(`profiles:list-updated`, () =>
  updateProfileList()
);
```

Reply:
sweet